vcpkg linux fails with cmake error  Unknown arguments specified`"CMAKE_BUILD_TYPE" "STREQUAL" "Debug" "OR" "Debug" "IN_LIST" "MAP_IMPORTED_CONFIG_Debug"`

I added `wxwidgets` to my `vcokg.json` then ran debug in VS Code

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
  "name": "combustion",
  "version": "1.0.1",
  "dependencies": [
    { "name": "wxwidgets",  "platform": "linux" }
  ]
}
```
### Description

```
[cmake] CMake Error at build/vcpkg_installed/x64-linux/share/wxwidgets/vcpkg-cmake-wrapper.cmake:40 (if):
[cmake]   if given arguments:
[cmake] 
[cmake]     "CMAKE_BUILD_TYPE" "STREQUAL" "Debug" "OR" "Debug" "IN_LIST" "MAP_IMPORTED_CONFIG_Debug"
[cmake] 
[cmake]   Unknown arguments specified
[cmake] Call Stack (most recent call first):
[cmake]   /home/jonny/Projects/vcpkg/scripts/buildsystems/vcpkg.cmake:783 (include)
[cmake]   CMakeLists.txt:31 (find_package)
[cmake] 
[cmake] 
[cmake] -- Configuring incomplete, errors occurred!
```


### Platform and version information

- wxWidgets version <!-- [e.g. 3.2.1] --> you are building: latest from vcpkg
- wxWidgets port <!-- [e.g. wxMSW, wxGTK, wxOSX] --> you are building: no idea
- OS <!-- [e.g. Windows 10, Ubuntu 22.10, macOS 15] --> and its version: ubuntu 22 lts
- Compiler <!-- [e.g. MSVS 2022, gcc 12.1] --> being used: gcc

--- a/ports/wxwidgets/vcpkg-cmake-wrapper.cmake
+++ b/ports/wxwidgets/vcpkg-cmake-wrapper.cmake
@@ -1,6 +1,7 @@
 cmake_policy(PUSH)
 cmake_policy(SET CMP0012 NEW)
 cmake_policy(SET CMP0054 NEW)
+cmake_policy(SET CMP0057 NEW)
 
 get_filename_component(_vcpkg_wx_root "${CMAKE_CURRENT_LIST_DIR}/../.." ABSOLUTE)
 set(wxWidgets_ROOT_DIR "${_vcpkg_wx_root}" CACHE INTERNAL "")

--- a/ports/wxwidgets/vcpkg.json
+++ b/ports/wxwidgets/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "wxwidgets",
   "version": "3.2.1",
-  "port-version": 1,
+  "port-version": 2,
   "description": [
     "Widget toolkit and tools library for creating graphical user interfaces (GUIs) for cross-platform applications. ",
     "Set WXWIDGETS_USE_STL in a custom triplet to build with the wxUSE_STL build option.",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7974,7 +7974,7 @@
     },
     "wxwidgets": {
       "baseline": "3.2.1",
-      "port-version": 1
+      "port-version": 2
     },
     "wyhash": {
       "baseline": "2022-10-12",

--- a/versions/w-/wxwidgets.json
+++ b/versions/w-/wxwidgets.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b7b94147f3dfd0c3a6f137121ef874159d2d9c0c",
+      "version": "3.2.1",
+      "port-version": 2
+    },
+    {
       "git-tree": "3a640144a98d7909a92ba363114f95bcaff554e5",
       "version": "3.2.1",
       "port-version": 1

